src/App.jsx

```diff
@@ -0,0 +1,38 @@
+import "./App.css";
+import {
+  Route,
+  createBrowserRouter,
+  createRoutesFromElements,
+  RouterProvider,
+} from "react-router-dom";
+import { Provider } from "react-redux";
+import Layout from "./routes/Layout";
+import Rockets from "./routes/Rockets";
+import Missions from "./components/Missions";
+import MyProfile from "./routes/MyProfile";
+import "bootstrap/dist/css/bootstrap.min.css";
+import store from "./redux/store";
+
+const router = createBrowserRouter(
+  createRoutesFromElements(
+    <Route
+      path="/"
+      element={<Layout />}
+      errorElement={<div>Error Element</div>}
+      ErrorBoundary={<div>Error Boundaries</div>}
+    >
+      <Route path="/" element={<Rockets />} />
+      <Route path="/missions" element={<Missions />} />
+      <Route path="/myProfile" element={<MyProfile />} />
+    </Route>
+  )
```

Comment:
- [ ] It's great that you have a blue text on the active page team ✅. But It's a common practice and provides an advantage when it comes to UX to indicate an active page with underlined text. Please add an underline on the active page.

![image](https://github.com/DilsherB/Space-Travelers-Hub/assets/69077061/233b4636-1afc-4403-b0ca-0614b642ad55)